docs/mesos/ManualInstallCalico.md

```diff
@@ -0,0 +1,65 @@
+# Manually Install the Calico Mesos Plugin
```

Comment:
"Calico Mesos plugin" is the official title of the tiny little binary we install, but this guide covers more components than that. Could use a better title to cover everything:

`Manually Add Calico Networking to Mesos` ?